stack.ghc-8.6.yaml

```diff
@@ -47,7 +47,7 @@ extra-deps:
   - splitmix-0.0.4
   - bv-sized-1.0.1
   - bitwise-1.0.0.1
-  - sbv-8.4
+  - sbv-8.7
```

Comment:
FYI, SBV 8.7 requires a _quite_ recent version of CVC4, and the bug I originally pulled in SBV 8.7 for has now been fixed a different way.

Reply:
Should we fall back on 8.6, then?

Reply:
So maybe I should relax this to 8.6?

Reply:
Well, I guess in a `stack.yaml` file it's not "relaxing" the version number, as it's not a lower bound. Maybe I should "downgrade" it to 8.6?

Reply:
Actually the ghc-8.8.yaml file has sbv-8.6, so it would be good for those to match.

Reply:
I think that's probably the right course of action for now.